purchase_request_to_rfq_order_approved/models/purchase_request.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+from openerp import models, api
+
+
+class PurchaseRequestLine(models.Model):
+    _inherit = "purchase.request.line"
+
+    @api.multi
+    @api.depends('purchase_lines.state', 'purchase_lines.order_id.state')
+    def _compute_purchase_state(self):
+        temp_purchase_state = \
+            super(PurchaseRequestLine, self)._compute_purchase_state()
+        for rec in self:
+            temp_purchase_state = False
```

Comment:
@gmeficent This statement results in many purchase request lines with state = False! Remove this.